FreeRTOS/FreeRTOS-Kernel
# Port Arm Cortex-M55

Description
-----------
Add Cortex-M55 support based on Cortex-M33 support.
M55 files extended only with MVE context save-restore mechanism. The same mechanism is used as in case of FPU on M33.
No assembly changed compared to M33, only `configENABLE_MVE` macro is added.
ARM_CM55_TFM port is also added.

Test Steps
-----------
TZ and MPU demos run from FreeRTOS/FreeRTOS repository.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/477


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
